app/views/welcome/_control_bar.html.erb

```diff
@@ -1,23 +1,23 @@
 <div class="header-control-bar">
   <% if current_page?(root_url) %>
```

Comment:
you don't have tests for any of these `if` statements about which buttons to display